### PR TITLE
Migrate to propshaft

### PR DESCRIPTION
**Pull request by segiddins**
This time, skip all bundlers and load raw css/js files

Alternative take on https://github.com/rubygems/rubygems.org/pull/3470 that avoids the bundlers that need a procfile & node

### CODE REVIEW

**Review by indirect — APPROVED** [review]
I'm happy to do something like this, including via rebasing and merging this PR, once we land something like #4396.

**Comment by martinemde** [comment]
Rebased on the importmaps in master now. I think this should be correct but I would appreciate a second pass because it's almost entirely different.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -13,9 +13,8 @@
     <link rel="fluid-icon" href="/fluid-icon.png"/>
     <link rel="search" type="application/opensearchdescription+xml" title="<%=t :title %>" href="/opensearch.xml">
     <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
-
-    <%= stylesheet_link_tag("application") %>
     <%= stylesheet_link_tag("tailwind", "data-turbo-track": "reload") %>
+    <%= stylesheet_link_tag("application", "preload_links_header": true) %>
```
Comment (martinemde) [inline app/views/layouts/application.html.erb:17]: I'm not sure about the order here, but tailwind gem installed it this way 🤷‍♂️

**Review by martinemde — APPROVED** [review]
I did the major rebase, so approving so that @segiddins can merge if he "approves" of my rebase.

**Comment by martinemde** [comment]
Something might not be right. This happened to me after clearing cache locally and it's not restoring.

Screenshot 2024-02-09 at 7 41 36 PM

**Comment by martinemde** [comment]
Ok, I was using this for dev again today and I had no problems. It solved many of the problems I was having and never caused any issues.

**Review by martinemde — APPROVED** [review]
I'm fine testing a deploy of this to staging to make sure all assets build. We can revert if they don't.

**Pull request merged**